ui/tests/integration/components/pki/pki-key-parameters-test.js

```diff
@@ -13,6 +13,7 @@ module('Integration | Component | pki-key-parameters', function (hooks) {
     this.store = this.owner.lookup('service:store');
     this.model = this.store.createRecord('pki/role');
     this.model.backend = 'pki';
+    [this.fields] = Object.values(this.model.fieldGroups.find((g) => g['Key parameters']));
```

Comment:
rather than hardcoding fields - decided to pull from the model so that if there are openapi changes they will be reflected in the test

though I think I'll move checking for these values/defaults to the pki role-form itself and generalize this component test when writing test for generating a key